index.html

```diff
@@ -0,0 +1,78 @@
+  <p class="content-text">Логотип BMW почали часто асоціювати із зображенням гвинта. Директор архіву BMW Group
+    Classic Фред Якобс запевняє, що насправді це легенда. Вона випливає з реклами, опублікованої у 1929 році, зображуючи
+    літак з написом BMW на гвинті в польоті. І дійсно, він став схожим на логотип.
+  </p>
+  <img alt="BMW advertise" class="content-image" loading="lazy" src="img/2.jpeg"/>
+  <p class="content-text">Завдання цієї реклами на самому початку світової економічної кризи полягало в просуванні
+    нового авіаційного двигуна, що випускався BMW за ліцензією Pratt & Whitney. Пропелер добре вписувався у рекламний
+    імідж молодої компанії, оскільки це підкреслювало її коріння та досвід у створенні авіамоторів.
+  </p>
+  <img alt="BMV engine on plane" class="content-image" loading="lazy" src="img/3.jpeg"/>
+  <p class="content-text">Насправді, спростовувати міф в компанії не поспішали. У 1942 році, BMW ще раз пов’язав
+    гвинт із символом своєї компанії. Стаття з’явилась у публікації BMW під назвою «Flugmotoren-Nachrichten» (Новини
+    авіаційного двигуна), яка підкріпила легенду про походження лого: в статті додали фото літака з назвою BMW на
+    гвинті.
+    Цей міф жив багато років і навіть сьогодні його часто пов’язують з сучасним логотипом бренду. «Ця інтерпретація була
+    поширена протягом 90 років, і згодом вона отримала певне право на існування», — додав Якобс.
+  </p>
+  <img alt="logo" class="content-image" loading="lazy" src="img/4.jpeg"/>
+  <h3 class="content-title">Що дійсно зображено на лого</h3>
+  <p class="content-text">Насправді ж, основа дизайну логотипа BMW — його кольорова гама. Білий і синій — кольори
+    штату Баварія в Німеччині — те, де народився бренд.
+    А ще ці кольори та навіть візерунки, схожі на ті, що частково зображені на логотипі BMW, можна побачити на прапорі
+    землі Баварія.
+  </p>
+  <img alt="Bavarian flag" class="content-image" loading="lazy" src="img/5.webp"/>
+  <p class="content-text"><em>Баварський прапор</em></p>
+  <h3 class="content-title">Але є законодавчий нюанс</h3>
+  <p class="content-text">Кольори в логотипі подібні до баварського прапору, але зображені вони у зворотному
+    порядку. Причиною цього став місцевий закон про товарні знаки того часу, який забороняв використання державних
+    гербів чи інших символів суверенітету на комерційних логотипах.
+  </p>
+  <h3 class="content-title">Як змінювався логотип протягом століття</h3>
+  <p class="content-text">Компанія BMW залишалася вірною своєму першому логотипу. У центрі завжди був той саме
+    візерунок та ті самі кольори. З роками змінювалися лише стилі: дизайнери намагалися не відставати від модних
+    тенденцій, додавати об’єм, кольори фону, змінювати шрифт тексту та товщину кільця.
+  </p>
+  <img alt="BMW logo from 1917 to 1933" class="content-image" loading="lazy" src="img/6-1.jpeg"/>
```

Comment:
Виглядає як добрий кандидат на 
```suggestion
<figure>
  <img alt="BMW logo from 1917 to 1933" class="content-image" loading="lazy" src="img/6-1.jpeg"/>
  <img alt="BMW logo from 1917 to 1933" class="content-image" loading="lazy" src="img/6-2.jpeg"/>
  <img alt="BMW logo from 1917 to 1933" class="content-image" loading="lazy" src="img/6-3.jpeg"/>
  <img alt="BMW logo from 1917 to 1933" class="content-image" loading="lazy" src="img/6-4.jpeg"/>
  <img alt="BMW logo from 1917 to 1933" class="content-image" loading="lazy" src="img/6-5.jpeg"/>
  <figcaption>
    <p class="content-text">Фото: <a href="https://www.bmw.com/en/automotive-life/bmw-logo-meaning-history1.html"><em>BMW</em></a>
    </p>
  </figcaption>
<figure>
```

Reply:
Так, дякую